packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -494,7 +494,6 @@ describe('postgresql', () => {
   })
 })
 
-// TODO remove .only
```

Comment:
There was an `only` we removed somewhere. I think it was directly below that comment, but it might have been somewhere else. It took us a while to figure out that's what was making the `lint` CI job failing.